### PR TITLE
Include cluster id whenever cluster name is used

### PR DESCRIPTION
**Pull request by SEQUOIIA**
Issue: https://github.com/dfds/cloudplatform/issues/1876

Before:
Screenshot 2023-08-17 at 1 25 36@2x
After:
Screenshot 2023-08-21 at 3 39 42@2x

---

Before:
Screenshot 2023-08-17 at 1 26 04@2x
After:
Screenshot 2023-08-17 at 1 18 20@2x

---

Before:
Screenshot 2023-08-17 at 1 26 18@2x
After:
Screenshot 2023-08-17 at 1 24 50@2x

### CODE REVIEW

File: src/src/pages/capabilities/KafkaCluster/index.js [hunk 1]
```diff
@@ -92,7 +92,7 @@ export default function KafkaCluster({ cluster, capabilityId }) {
 
   return (
     <PageSection
-      headline={`Kafka Topics (${cluster.name.toLocaleLowerCase()})`}
+      headline={` `} headlineChildren={(<div style={{display: 'flex', alignItems: 'center'}}><span style={{}}>Kafka Topics ({cluster.name.toLocaleLowerCase()}  -&nbsp;</span><span style={{fontSize: '0.8em', color: '#c6c7c5', top: '1px', position: 'relative'}}>{cluster.id}</span>)</div>)}
```
Comment (SEQUOIIA) [inline src/src/pages/capabilities/KafkaCluster/index.js:95]: Could put the styling into its own separate file. Dunno.

**Comment by kralle333** [comment]
Has a short chat with Emil and I had the following comments:

- I do not believe that we fully solve the problem that spawned this [task](https://dfds.slack.com/archives/C9948TVRC/p1691405152545489) - I suggested we somehow display the parameter path + link to aws documentation for what parameter store is.
- I think the cluster id shown in Kafka Topics should be removed or at least the color should be made darker to not blend in with the background. We could not agree on this so for now it will stay like this.

File: src/src/pages/capabilities/KafkaCluster/index.js (hunk 1 above)
Comment (kralle333) [reply src/src/pages/capabilities/KafkaCluster/index.js:95]: please do :)

**Review by kralle333 — CHANGES_REQUESTED** [review]

File: src/src/pages/capabilities/KafkaCluster/index.js (hunk 1 above)
Comment (andreasfrisch) [reply src/src/pages/capabilities/KafkaCluster/index.js:95]: I believe having the style in the same file is preferable (and the react-way), but it should also preferably not be inline

**Review by andreasfrisch — COMMENTED** [review]
I think we should agree on how to do styling in our react world.
Personally I prefer if the styles are defined in the same file as the component without being inline styling.
Sometimes inline styling has benefits, so I do not dislike it, but it often quickly becomes very hard to read.
'Remote styling' was the way to do things previously, but it just hide interesting details from the reader on first visit and I do not like that.

**Comment by kralle333** [comment]
> 

As the least experienced with React of us three, i'll leave it up to you and @SEQUOIIA :)

**Comment by SEQUOIIA** [comment]
> I think we should agree on how to do styling in our react world. Personally I prefer if the styles are defined in the same file as the component without being inline styling. Sometimes inline styling has benefits, so I do not dislike it, but it often quickly becomes very hard to read. 'Remote styling' was the way to do things previously, but it just hide interesting details from the reader on first visit and I do not like that.

I'd be happy with styling defined in the same file as the component. Just need to figure out a nice way to do it that works with React/JSX.

**Comment by SEQUOIIA** [comment]
Screenshot 2023-08-21 at 3 39 42@2x
Changed it to be a bit easier to spot

**Review by kralle333 — APPROVED** [review]
Happy with the changes and so is the person who spawned this task.

**Pull request merged**